### PR TITLE
Update mixer concept page

### PR DESCRIPTION
**Pull request by krisdale**
- Update concept page to include a comparative overview of mixer and mixin
- Added images to illustrated differences between mixer and mixin
- Add related links
- Corrected label for deploy-at-scale.rst so that it could be linked from mixer concept page

Signed-off-by: Kristal Dale <kristal.dale@intel.com>

### CODE REVIEW

**Comment by krisdale** [comment]
@bktan8 Here is the updated mixer content we have collaborated on. Please provide final review here. 

@mvincerx @intelkevinputnam - please review the updated concept. Feedback is appreciated wrt bkms, opportunities to make this an updated example for concepts.

Thanks!

**Comment by intelkevinputnam** [comment]
@mvincerx When you get a chance, can you take a look at the diff between my commit and Kris'? If you agree, we can leave the changes in. Otherwise we can revert to Kris'.

https://github.com/clearlinux/clear-linux-documentation/pull/329/commits/5dcc51337bb47a9c66d3aaf69d5ef27616ecd8d8

**Comment by intelkevinputnam** [comment]
@mvincerx Thanks for the feedback. I think I've addressed all of it.

**Comment by intelkevinputnam** [comment]
@bktan8 Can you review?

File: source/clear-linux/concepts/mixer-about.rst [hunk 1]
```diff
@@ -3,21 +3,80 @@
...
-using swupd. You will also need a working knowledge of
-:abbr:`RPMs (RPM Package Manager files)` and how |CL| bundles work. For a
-detailed guide on how to create a |CL| mix, visit :ref:`mixer`.
+|CL-ATTR| is a powerful, modular, and customizable OS. Upstream |CL| offers
+many images to support different environments and use-cases.  There are
+hundreds of bundles that will meet most, if not all, of your OS and software
+needs.
+
+However, if you need additional customization or content, |CL| provides the
+mixer tool. We provide two different ways to develop and extend the OS, depending on your needs:
```
Comment (bktan8) [inline source/clear-linux/concepts/mixer-about.rst]: Let's change "However, if you need additional customization or content, |CL| provides the
mixer tool. We provide two different ways to develop and extend the OS, depending on your needs:" to "However, if you need additional customization or content, |CL| provides the
mixer tool. Depending on your needs, the mixer tool allows you to:"  

"develop and extend the OS" seems a little strong for me.

Comment (mvincerx) [reply source/clear-linux/concepts/mixer-about.rst]: OK with me.

Comment (intelkevinputnam) [reply source/clear-linux/concepts/mixer-about.rst]: @bktan8 @mvincerx Updated.

**Comment by intelkevinputnam** [comment]
@bktan8 @mvincerx Thanks for the feedback. Looks like we're nearly there.

**Comment by mvincerx** [comment]
LGTM. Thanks @intelkevinputnam, @bktan8 for the review.

**Pull request merged**